MANUAL.txt

```diff
@@ -1415,13 +1415,17 @@ results only for basic math, usually you will want to use
 `--gladtex`
 
 :   Enclose TeX math in `<eq>` tags in HTML output.  The resulting HTML
-    can then be processed by [GladTeX] to produce images of the typeset
-    formulas and an HTML file with links to these images.
-    So, the procedure is:
+    can then be processed by [GladTeX] to produce SVG images of the typeset
+    formulas and an HTML file with these images embedded.
 
         pandoc -s --gladtex input.md -o myfile.htex
-        gladtex -d myfile-images myfile.htex
-        # produces myfile.html and images in myfile-images
+        gladtex -d image_dir myfile.htex
+        # produces myfile.html and images in image_dir
+
+    Alternatively, you can use GladTeX as a Pandoc filter:
+
+        pandoc -t json input.md | gladtex -d image_dir -P - \
+            |pandoc -f json -t html -o myfile.html -
```

Comment:
This is cool, I had no idea gladtex could be used this way!
I'd suggest formatting like
```
pandoc -t json input.md | gladtex -d image_dir -P - | \
    pandoc -f json -t html -o myfile.html
```
(You don't need the `-` for stdin on the last pandoc invocation, and I think it's clearer with the pipe on the line before.  If you'd rather have the pipe on the second line, how about leaving a space after it?)